### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by Dwwwarfff**
Is the test correct?

### CODE REVIEW

File: task2.py [hunk 1]
```diff
@@ -50,6 +53,6 @@ def quantity_of_words(queries):
         'сериалы про спорт',
     ]
 
-
     for key, value in quantity_of_words(queries).items():
         print(f"Search queries containing {key} word(s):{round(percent(len(queries), value), 2)}%")
```
Comment (Dwwwarfff) [inline task2.py]: I probably should add more comments here

File: task2.py [hunk 2]
```diff
@@ -22,22 +22,23 @@
 """
 
 
-def percent(tota, num):
-    return (100 * num) / tota
+def percent(total, num):
     # Считаем проценты
+    return (100 * num) / total
```
Comment (Dwwwarfff) [inline task2.py:27]: Leave a comment that we believe, that total != 0

File: task2.py [hunk 3]
```diff
@@ -22,22 +22,23 @@
 """
 
 
-def percent(tota, num):
-    return (100 * num) / tota
+def percent(total, num):
```
Comment (Dwwwarfff) [inline task2.py]: Also make test(s) for this function too

File: task2.py (hunk 2 above)
Comment (Dwwwarfff) [reply task2.py:27]: So I tried a script where total = 0 - I added an if construction, but it doesn't work. If queries = [' '], than len(queries) (total for us here) = 1, if queries = [], than the program doesn't start. So there won't be an exepsion

File: task2.py [hunk 4]
```diff
@@ -22,7 +22,7 @@
 """
 
 
-def percent(total, num):
+def percent_culc(total, num):
```
Comment (NPeganov) [inline task2.py]: percent_cAlc

**Review by NPeganov — APPROVED** [review]

**Pull request merged**